cmd/storage-interface.go

```diff
@@ -41,8 +41,9 @@ type StorageAPI interface {
 	// File operations.
 	ListDir(volume, dirPath string, count int) ([]string, error)
 	ReadFile(volume string, path string, offset int64, buf []byte, verifier *BitrotVerifier) (n int64, err error)
-	PrepareFile(volume string, path string, len int64) (err error)
 	AppendFile(volume string, path string, buf []byte) (err error)
+	CreateFile(volume, path string, size int64, reader io.Reader) error
```

Comment:
Let's increment storage RPC version so nodes won't communicate unless all of them are updated